<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpellBase</class>
 <widget class="QWidget" name="SpellBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>465</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Term Explorer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QComboBox" name="expTypeCMB"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="baseWordLE">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Match</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="caseSensCB">
         <property name="text">
          <string>Case</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="diacSensCB">
         <property name="text">
          <string>Accents</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="stemLangCMB"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="expandPB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>&amp;Expand </string>
         </property>
         <property name="shortcut">
          <string>Alt+E</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="dismissPB">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>&amp;Close</string>
         </property>
         <property name="shortcut">
          <string>Alt+C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="statsLBL">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>No db info.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="resTW">
       <property name="rowCount">
        <number>2</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>baseWordLE</tabstop>
  <tabstop>expandPB</tabstop>
  <tabstop>dismissPB</tabstop>
  <tabstop>stemLangCMB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
